_posts/2017-09-15-comment-ceder-une-startup-detat.md

```diff
@@ -0,0 +1,40 @@
+
+Transférer un service public numérique d’une petite organisation de culture agile vers une grande organisation de culture contractuelle va donc naturellement provoquer un désir de « retour à la normale » vers le cadre contractuel connu. Mais mettre en risque les pratiques qui ont permis de construire le service, c’est mettre en péril le service lui-même ! Facebook pourrait-il encore être le premier réseau social mondial s’il était en maintenance dans une SSII&nbsp;?
+
+## Éloge de la différence
+
+À la vérité, l’immense majorité des DSI souhaite aujourd’hui intégrer de nouvelles manières de faire, sans toujours trouver le chemin qui leur permette de s’extraire d’habitudes ancrées dans l’organisation.
+
+Accueillir une équipe travaillant autrement est donc une opportunité unique d’initier un tel changement, sans rupture. En effet, rien n’empêche la DSI de fonctionner dans ses habitudes et son organisation classique MOA/MOE/Exploitation, et d’héberger dans le même temps des équipes intégrées, de l’utilisateur à l’exploitant, en passant par le développeur. 
+
+Les bénéfices sont triples :
+
+* continuer à enrichir le produit qui a montré l'efficacité de l’organisation intégrée jusqu'ici;
+* enrichir l’offre de service de la DSI, en proposant un nouveau guichet à ses usagers : l’offre agile ;
+* permettre à d’autres équipes existantes d’envisager basculer [dans ce mode](https://beta.gouv.fr/2016/11/28/equipes-autonomes.html) si elles le souhaitent : la présence d’une première équipe agile permet de lever les barrières et d’aider les informaticiens volontaires à basculer, notamment ceux qui touchent les limites du système contractuel, là où cela fonctionne le moins bien.
+
+## Une passation en douceur
+
+Pour sécuriser une passation de service, nous proposons généralement un schéma fondé sur le volontariat :
+Le DSI lance un appel à vocation pour des opérationnels souhaitant s’investir à la fois dans le service et dans les pratiques : _« Vous êtes développeur informatique ou ingénieur système ? Vous souhaitez travailler selon les principes des méthodes agiles dans une équipe devOps intégrée ? Le champ du [domaine de la Startup], et des technologies comme [outils de la Startup] vous intéresse ? Contactez-nous ! »_.
+
+L’incubateur accueille ces personnes pour 3 à 9 mois de travail en binôme avec l’équipe historique de la startup qui l’acculture à ses méthodes : dialogue usager, pilotage par l’impact, itérations fixes, tests automatisés, remaniement de code, rituels d’équipe…
+Transfert final dans la DSI une fois l’acculturation des collaborateurs terminée. Des membres de l’équipe historique qui souhaiteraient accompagner le service dans la durée peuvent également être embauchés à l’occasion.
```

Comment:
> Transfert final dans la DSI une fois l’acculturation des collaborateurs terminée.

La phrase n'a pas de verbe, ce qui me fait butter dessus à la lecture. Par ailleurs, le lien logique entre les trois blocs n'est pas évident. Peut être qu'on pourrait ajouter quelques mots de liaison ? 

> D'abord, la DSI lance un appel...
> Ensuite, l'incubateur accueille...
> Enfin, le service est transféré dans la DSI une fois...

Reply:
OK
L’incubateur accueille ces personnes pour 3 à 9 mois de travail en binôme avec l’équipe historique de la startup qui l’acculture à ses méthodes : dialogue usager, pilotage par l’impact, itérations fixes, tests automatisés, remaniement de code, rituels d’équipe…
Lors de ce passage, nous encourageons les membres de l’équipe historique qui souhaiteraient accompagner le service dans la durée à se faire embaucher à l’occasion.